doc/release/README.tasks

```diff
@@ -57,15 +57,14 @@ Chapel's default tasking layer implementation for most targets is based
 on the Qthreads user-level threading package from Sandia National Labs.
 This provides a lightweight implementation of Chapel tasking and will
 also ultimately provide an optimized implementation of sync variables.
-Qthreads tasking is the default for all targets except Intel KNC and
-Cygwin.  To use qthreads tasking, please take the following steps:
+To use qthreads tasking, please take the following steps:
 
 1) Ensure that the environment variable CHPL_HOME points to the
    top-level Chapel directory, as always.
 
 2) Set up your environment to use Qthreads:
 
-      ensure CHPL_TASKS is not set
+      ensure CHPL_TASKS is not set (if qthreads is the default)
```

Comment:
yea, I thought about doing something similar especially since the original instructions implied that qthreads was always the default if not set. I ended up going with this because I couldn't get the line length under 72 characters, which is kind of a lame reason but I think I'll leave it as is unless you think otherwise.